### DISCUSSION TITLE
Creating a MachineUser

### DISCUSSION THREAD

**Discussion by PhillypHenning** [bitovi/bitops#217] [General]
@mickmcgrath13 
some options:
[bitops@bitovi.com](mailto:bitops@bitovi.com)
[docker@bitovi.com](mailto:docker@bitovi.com)
[machine_user@bitovi.com](mailto:machine_user@bitovi.com)
[bot@bitovi.com](mailto:bot@bitovi.com)

that has broad implications potentially (would need a much wider discussion as Bitovi has lots of possible needs for such a user)

**Comment by mickmcgrath13** [comment]
this is done, btw.
PR: https://github.com/bitovi/bitops/commit/3e55f09ed59abdcd104d295eb4debb27f327ba76
Machine user (dockerhub user): `osbitovi`
